python/py-fipy/Portfile

```diff
@@ -18,30 +20,32 @@ long_description    FiPy is an object oriented, partial differential \
                     for viewing.
 
 homepage            http://www.ctcms.nist.gov/fipy/
-master_sites        http://www.ctcms.nist.gov/fipy/download/
-distname            FiPy-${version}
 
-checksums           rmd160  19891efaf2371fce3baa0acf0f4360116d7dc854 \
-                    sha256  54a4bc2ed6562bd402890ddd257220cf092438db1941427035412813415bd0c7
+checksums           rmd160  0454eb328b6ff315e53cb3a9f38494a6c5aa7f26 \
+                    sha256  a71cadc7717f464c6daf29df7b4c9ce10bc1753e0f1b397124078b67182ef467 \
+                    size    10331174
 
-python.versions     27
+python.versions     27 36
```

Comment:
Why are you adding py36 and not py37? The latest release version of python 3 is preferred because it will stay around longer.

Reply:
Well, because I have tested Fipy under Python 3 only in my Python 3.6 installation. Thanks for your review; I added py37 to the Portfile.